### PR TITLE
Improve API docs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: task-ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+## Module Overview
```
Comment (shafreenAnfar) [inline task-ballerina/Module.md:1]: Just `Overview `

File: task-ballerina/Module.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+- One-time Job Execution
+- Frequency-based Job Execution 
+
+#### One-time Job Execution
+
+This API provides the functionality to schedule a job at a specified date.
+
+The following code snippet shows how to schedule a one-time job.
+
```
Comment (shafreenAnfar) [inline task-ballerina/Module.md:19]: Additional space.

File: task-ballerina/Package.md [hunk 3]
```diff
@@ -1,94 +1,14 @@
...
-    second: 50.52,
-    timeAbbrev: "Asia/Colombo",
-    utcOffset: zoneOffset
-};
-
-task:Error|task:JobId result = task:scheduleJobRecurByFrequency(new Job("Hi"), 2.5, maxCount = 10, startTime = time);
-```
-For an example on the usage of the `scheduleJobRecurByFrequency`, see the [Task Frequency Job Execution Example](https://ballerina.io/learn/by-example/task-frequency-job-execution.html).
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```
Comment (MadhukaHarith92) [inline task-ballerina/Package.md:12]:
```suggestion
- Discuss code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```

File: task-ballerina/Package.md [hunk 4]
```diff
@@ -1,94 +1,14 @@
...
+- Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
+- Post all technical questions on Stack Overflow with the #ballerina tag.
```
Comment (MadhukaHarith92) [inline task-ballerina/Package.md:14]: Add the link here.

File: task-ballerina/Package.md [hunk 5]
```diff
@@ -1,94 +1,14 @@
 ## Package Overview
 
-This package provides the functionality to schedule a Ballerina job and manages the execution of Ballerina jobs either once or periodically.
+This `task` library is one of the standard libraries of <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```
Comment (daneshk) [inline task-ballerina/Package.md:3]:
```suggestion
This Task library is one of the standard libraries of <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

Comment (MadhukaHarith92) [reply task-ballerina/Package.md:3]: Should be "The Task library"

File: task-ballerina/Module.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
+## Module Overview
+
+This module provides the functionality to schedule a Ballerina job either once or periodically and to manage the execution of those jobs.
+
+### Jobs and Scheduling
+
+Every scheduling job in Ballerina needs to be represented by a Job object. Therefore, You need to create a jobs class with your custom logic to execute it when the task is triggered.
```
Comment (MadhukaHarith92) [inline task-ballerina/Module.md:7]:
```suggestion
Every scheduling job in Ballerina needs to be represented by a Job object. Therefore, a job class with your custom logic needs to be created to execute it when the task is triggered.
```

Comment (MadhukaHarith92) [reply task-ballerina/Module.md:7]: Please avoid using "You", "We", etc.

File: task-ballerina/Module.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+The Task package has the following two scheduling systems to schedule the job:
+
+- One-time Job Execution
+- Frequency-based Job Execution 
+
+#### One-time Job Execution
+
+This API provides the functionality to schedule a job at a specified date.
```
Comment (MadhukaHarith92) [inline task-ballerina/Module.md:16]: Correct me if I'm wrong. Is it "specified date" or "specified time"?

File: task-ballerina/Module.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+};
+
+task:JobId result = check task:scheduleOneTimeJob(new Job("Hi"), time);
+```
+
+##### Frequency-based Job Execution
+
+This API provides the functionality to schedule jobs on a specific interval either once or periodically by configuring the configuration such as start time, end time, and maximum count.
+
+The following code snippet shows how to schedule a recurrence job by using this API.
```
Comment (MadhukaHarith92) [inline task-ballerina/Module.md:59]: `recurrence` or `recurring`?

Comment (kalaiyarasiganeshalingam) [reply task-ballerina/Module.md]: Changed to `recurring`

File: task-ballerina/Module.md (hunk 7 above)
Comment (kalaiyarasiganeshalingam) [reply task-ballerina/Module.md]: Changed to `time`

File: README.md [hunk 9]
```diff
@@ -11,7 +11,7 @@ This `task` library is one of the standard libraries of <a target="_blank" href=
 
 This package provides the functionality to configure and manage one-time or periodic jobs.
 
-For more information on all the operations supported by this package, go to [The Task Package](https://ballerina.io/learn/api-docs/ballerina/task/).
+For more information on all the operations supported by this package, go to [The Task Package](https://docs.central.ballerina.io/ballerina/task/latest).
```
Comment (praneesha) [inline README.md:14]:
```suggestion
For more information on all the operations supported by this package, go to the [`task` package](https://docs.central.ballerina.io/ballerina/task/latest).
```

File: README.md [hunk 10]
```diff
@@ -11,7 +11,7 @@ This `task` library is one of the standard libraries of <a target="_blank" href=
...
 
 For a quick sample on demonstrating the usage see [Ballerina By Example](https://ballerina.io/learn/by-example/).
```
Comment (praneesha) [inline README.md:16]:
```suggestion
For a quick sample on demonstrating the usage, see [Ballerina By Example](https://ballerina.io/learn/by-example/).
```

File: task-ballerina/Module.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+This module provides the functionality to schedule a Ballerina job either once or periodically and to manage the execution of those jobs.
+
+### Jobs and Scheduling
+
+Every scheduling job in Ballerina needs to be represented by a Job object. Therefore, a job class with your custom logic needs to be created to execute it when the task is triggered.
+
+The Task package has the following two scheduling systems to schedule the job:
+
+- One-time Job Execution
```
Comment (praneesha) [inline task-ballerina/Module.md:11]:
```suggestion
- One-time job execution
```

File: task-ballerina/Package.md [hunk 12]
```diff
@@ -1,94 +1,14 @@
 ## Package Overview
 
-This package provides the functionality to schedule a Ballerina job and manages the execution of Ballerina jobs either once or periodically.
+The Task library is one of the standard libraries of <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```
Comment (praneesha) [inline task-ballerina/Package.md:3]:
```suggestion
The `task` library is one of the standard libraries of <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

File: task-ballerina/Module.md (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply task-ballerina/Module.md:19]: Removed in 71fb824a5ad4b1f99955aac7e6b2da30d82fdb61

**Review by praneesha — APPROVED** [review]

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Pull request merged**